scannerazure.properties

```diff
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-azure-devops/
 issueTrackerUrl=https://jira.sonarsource.com/browse/VSTS/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-vsts
-archivedVersions=4.7,4.7.1,4.7.2,4.8,4.8.1,4.9,4.10,4.11,4.12,4.16,4.17,4.18,4.19,4.20,4.21,4.22,4.23,4.23.1,5.0.0,5.0.1,5.1.1,5.2.0,5.3.0,5.4.0,5.5.0,5.6.0,5.6.1,5.7.0,5.8.0,5.9.0
-publicVersions=5.10.0
+archivedVersions=4.7,4.7.1,4.7.2,4.8,4.8.1,4.9,4.10,4.11,4.12,4.16,4.17,4.18,4.19,4.20,4.21,4.22,4.23,4.23.1,5.0.0,5.0.1,5.1.1,5.2.0,5.3.0,5.4.0,5.5.0,5.6.0,5.6.1,5.7.0,5.8.0,5.9.0,5.10.0
+publicVersions=5.11.1
+
+5.11.1.description=Azure DevOps extension is compatible with SonarQube 10.0
```

Comment:
Please include 5.11.0 in this same PR so that you don't have to worry about the other one. (it's about bumping S4NET to 5.11.0)

Thanks!

Reply:
Do you have the related ticket number for the changeLog link?

Reply:
done